tests/upgrades/test_capsule.py

```diff
@@ -181,24 +182,26 @@ def test_post_user_scenario_capsule_sync_yum_repo(self, request):
         request.addfinalizer(lambda: cleanup(content_view, repo, product))
         repo_name = gen_string('alpha')
         rpm_name = rpm2.split('/')[-1]
-        org = entities.Organization().search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
+        org = default_sat.api.Organization().search(query={'search': f'name="{DEFAULT_ORG}"'})[0]
```

Comment:
could use the default_org fixture here instead, right?

Reply:
Again great catch, yep looks like it.

Reply:
There was one other use in this file, updated both.